src/vue/forms/PreIssuanceForm.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <form
+    class="app__form issuance-form"
+    @submit.prevent="submit"
+  >
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <file-field
+          :label="'issuance.pre-issuance-lbl' | globalize"
+          :note="'issuance.file-type-lbl' | globalize"
+          accept=".iss"
+          v-model="documents.preIssuance"
```

Comment:
`documents` object actually contains a single field: `preIssuance`. why do we ever need this `documents` object?

Reply:
Resolved, updated